upup/pkg/fi/cloudup/gcetasks/instancetemplate.go

```diff
@@ -390,6 +402,7 @@ type terraformInstanceTemplate struct {
 }
 
 type terraformInstanceCommon struct {
+	InternalOnly          bool                          `json:"internal_only"`
```

Comment:
I don't see this field in the terraform reference: https://www.terraform.io/docs/providers/google/r/compute_instance.html  I suspect we just need to configure the `terraformNetworkInterface` without the `access_config`, as you did above